Cargo.toml

```diff
@@ -31,7 +31,7 @@ fedimint-testing = { git = "https://github.com/m1sterc001guy/fedimint", rev = "2
 devimint = { git = "https://github.com/m1sterc001guy/fedimint", rev = "2541b9705aefde724d55683aa46fb6997d312d7e" }
 aead = { git = "https://github.com/m1sterc001guy/fedimint", rev = "2541b9705aefde724d55683aa46fb6997d312d7e" }
 tbs = { git = "https://github.com/m1sterc001guy/fedimint", rev = "2541b9705aefde724d55683aa46fb6997d312d7e" }
-threshold_crypto = { git = "https://github.com/fedimint/threshold_crypto" }
+threshold_crypto = { git = "https://github.com/fedimint/threshold_crypto", package = "fedimint-threshold-crypto", version = "0.2.1" }
```

Comment:
Does this need to be 0.2.1 or can it be 0.3.0?

Reply:
![image](https://github.com/m1sterc001guy/roastr/assets/133234413/01629612-ebcc-4b0a-94ce-06b1cb5fc794)
Looks like 0.2.1 was the last crates.io version. I'll file a bug on that repo.
https://crates.io/crates/fedimint-threshold-crypto